docs/adr/0008-split-bindings-to-modules-automatically.md

```diff
@@ -0,0 +1,64 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed superseded by [gogo](0008-split-bindings-to-modules-automatically.md)]
+* Deciders: []
+* Date: [2022-07-11 when the decision was last updated]
+
+## Context and Problem Statement
+
+In his current state, uniffi only allow to import any generated code to a single file. This is embarassing because in languages used to make bindings, the user has to import submodules.
+
```

Comment:
I have a related question here, are you working with 1 component/UDL file or multiple at once?  I've thought about having UniFFI generate a multi-module directory structure, but that's because I'm working on a multi component project that has multiple UDL files.  Whenever I've thought about doing this, it's been linked to having `uniffi-bindgen` work on multiple UDL files at once which has several potential benefits.

Reply:
I doubt. I mentioned two ways in the file in order to argue on.